### PR TITLE
[8.0] [ADD] Add new module purchase_order_line_variant_description

### PR DESCRIPTION
**Pull request by hurrinico**
This module is based on https://github.com/OCA/sale-workflow/tree/8.0/sale_order_line_variant_description and allow to use sale description on the purchase order lines as their names

### CODE REVIEW

**Comment by azmimr67** [comment]
Runbot tested :+1:

File: purchase_order_line_variant_description/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015 Alex Comba - Agile Business Group
```
Comment (eLBati) [inline purchase_order_line_variant_description/__init__.py]: Alex or Nico?

File: purchase_order_line_variant_description/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2015 Nicola Malcontenti - Agile Business Group
```
Comment (eLBati) [inline purchase_order_line_variant_description/__openerp__.py]: 2016?

File: purchase_order_line_variant_description/models/purchase_order_line.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# © 2015 Nicola Malcontenti - Agile Business Group
```
Comment (eLBati) [inline purchase_order_line_variant_description/models/purchase_order_line.py]: 2016?

**Comment by eLBati** [comment]
Some remarks, then :+1:

**Comment by hurrinico** [comment]
@eLBati Fixed

File: purchase_order_line_variant_description/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+{
+    'name': "Purchase order line variant description",
+    'version': '8.0.1.0.0',
+    'category': 'Purchase Management',
+    'author': "Agile Business Group, Odoo Community Association (OCA)",
+    'website': 'http://www.agilebg.com',
+    'license': 'AGPL-3',
+    "depends": [
+        'sale_order_line_variant_description',
```
Comment (tafaRU) [inline purchase_order_line_variant_description/__openerp__.py:13]: I think it's not needed so you can remove it.

File: purchase_order_line_variant_description/models/purchase_order_line.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            qty=qty, uom_id=uom_id,
+            partner_id=partner_id, date_order=date_order,
+            fiscal_position_id=fiscal_position_id,
+            date_planned=date_planned, name=name, price_unit=price_unit,
+            state=state)
+        if product_id:
+            product_obj = self.env['product.product']
+            lang = self.env['res.partner'].browse(partner_id).lang
+            product = product_obj.with_context(lang=lang).browse(product_id)
+            if product.variant_description_sale:
```
Comment (tafaRU) [inline purchase_order_line_variant_description/models/purchase_order_line.py:28]: Shouldn't be `description_purchase`?

File: purchase_order_line_variant_description/models/purchase_order_line.py [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+                if 'value' not in res:
+                    res['value'] = {}
+                res['value']['name'] = product.variant_description_sale
```
Comment (tafaRU) [inline purchase_order_line_variant_description/models/purchase_order_line.py:31]: same as above.

**Review by tafaRU — CHANGES_REQUESTED** [review]

File: purchase_order_line_variant_description/models/purchase_order_line.py (hunk 6 above)
Comment (hurrinico) [reply purchase_order_line_variant_description/models/purchase_order_line.py:31]: This Module is based on sale_order_line_variant_description as description says, and use [this](https://github.com/OCA/sale-workflow/blob/8.0/sale_order_line_variant_description/models/product_product.py#L12) field.
I'm going to check travis error now

**Comment by gurneyalex** [comment]
I just merged #529 which fixes the travis build of 8.0. Can you rebase?

**Pull request closed without merging**